packages/evm/src/utils/helpers.ts

```diff
@@ -25,9 +25,13 @@ export const createERCDepositData = (
   let recipientAddressInBytes;
   if (utils.isAddress(recipientAddress)) {
     recipientAddressInBytes = getEVMRecipientAddressInBytes(recipientAddress);
-  } else {
+  } else if (parachainId) {
     recipientAddressInBytes = getSubstrateRecipientAddressInBytes(recipientAddress, parachainId);
+  } else {
+    const hexAddress = addressToHex(recipientAddress, recipientAddress.length);
```

Comment:
this can be achieved using `hexlify(toUtf8Bytes(recipientAddress))` no need for `addressToHex` furthermore, to decode a bitcoin address the functions `fromBech32` and `fromBase58Check` from the `bitcoinjs-lib` can be used